receiver/mongodbatlasreceiver/receiver.go

```diff
@@ -98,21 +99,30 @@ func (s *receiver) poll(ctx context.Context, time timeconstraints) error {
 			return fmt.Errorf("error retrieving projects: %w", err)
 		}
 		for _, project := range projects {
+			nodeClusterMap, err := s.getNodeClusterNameMap(ctx, project.ID)
+			if err != nil {
+				return fmt.Errorf("error collecting clusters from project %s: %w", project.ID, err)
+			}
+
 			processes, err := s.client.Processes(ctx, project.ID)
 			if err != nil {
-				return fmt.Errorf("error retrieving MongoDB Atlas processes: %w", err)
+				return fmt.Errorf("error retrieving MongoDB Atlas processes for project %s: %w", project.ID, err)
 			}
 			for _, process := range processes {
+				clusterName := ""
+				if v, ok := nodeClusterMap[process.UserAlias]; ok {
+					clusterName = v
+				}
```

Comment:
This has the same effect.
```suggestion
				clusterName := nodeClusterMap[process.UserAlias]
```

Reply:
I got wrapped up in trying to prevent nils, forgot the zero-value behavior of the maps in golang when the key isn't present.